packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -119,17 +119,53 @@ class _ToolbarContainerLayout extends SingleChildLayoutDelegate {
 ///
 /// An app bar consists of a toolbar and potentially other widgets, such as a
 /// [TabBar] and a [FlexibleSpaceBar]. App bars typically expose one or more
-/// common actions with [IconButton]s which are optionally followed by a
+/// common [actions] with [IconButton]s which are optionally followed by a
```

Comment:
Added `(sometimes called the "overflow menu")`.

At some point we should probably do the folding into the menu thing automatically.